app/src/main/AndroidManifest.xml

```diff
@@ -9,7 +9,7 @@
         android:roundIcon="@mipmap/ic_launcher_round"
         android:supportsRtl="true"
         android:theme="@style/AppTheme">
-        <activity android:name=".MainActivity">
+        <activity android:name=".ui.meme_it_up.MemeItUpActivity">
```

Comment:
do not use **_** to separate words in packages, use **.**

Reply:
done, thanks